src/ott/tools/soft_sort.py

```diff
@@ -478,47 +477,46 @@ def multiv_cdf_quantile_maps(
   the other for the quantiles map.
 
   Args:
-    inputs: 2D array of `[n, d]` vectors.
-    targets: 2D array of `[m,d]` vectors. If not passed explicitly, the random
-      sampler specified in `target_sampled` is used to create these samples from
-      reference measure.
-    target_sampler: When ``targets`` is not passed, Callable that takes a
-      ``key`` and shape specified by ``target_num_samples`` to sample a target
-      points.
-    key: `rng` key used in ``target_sampler``
-    target_num_samples: number of points generated in the target distribution.
-    cost_fn: cost function used to compare ``inputs`` and ``targets``. Passed on
-      to instantiate a {class}`~ott.geometry.pointcloud.PointCloud` object. This
-      will default to squared-Euclidean distance.
+    inputs: 2D array of :math:`[n, d]` vectors.
+    target_sampler: Callable that takes a ``key`` and ``[m,d]`` shape Tuple.
+      ``m`` is passed on as ``target_num_samples``, dimension ``d`` is inferred
+      directly from the shape passed in ``inputs``. This is assumed by default
+      to be :func:`jax.random.uniform`, and could be any other random sampler
+      properly wrapped to have the signature above.
+    key: rng key used by ``target_sampler``
+    num_target_samples: number ``m`` of points generated in the target
+      distribution.
+    cost_fn: :class:`~ott.geometry.costs.CostFn` object, used to compare
```

Comment:
I wouldn't use the :class:\`~ott.geometry.costs.CostFn\`, it's already in the type annotation.